### PR TITLE
Fix test for biggus 0.14.

### PR DESCRIPTION
**Pull request by pp-mo**
Tiny fix needed as private element got moved.

### CODE REVIEW

File: lib/iris/tests/unit/cube/test_Cube__operators.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import numpy as np
 import numpy.ma as ma
 import biggus
+from biggus._init import _Elementwise as Elementwise
```
Comment (marqh) [inline lib/iris/tests/unit/cube/test_Cube__operators.py]: i do not like importing a private class definition and changing its name to remove the leading underscore

**Comment by marqh** [comment]
thanks @pp-mo

**Comment by QuLogic** [comment]
Does this not mean the minimum dependency needs to be updated?

**Comment by pp-mo** [comment]
> Does this not mean the minimum dependency needs to be updated?

No, this is not an API change :  The _Elementwise class was, and remains, a private element.

**Comment by QuLogic** [comment]
But there is no `biggus._init` in Biggus 0.13, though I guess it _is_ only a test...

**Comment by ajdawson** [comment]
You could use a try/except to avoid this, e.g.:

```python
try:
    from biggus._init import _Elementwise
except ImportError:
    _Elementwise = biggus._Elementwise
```

**Pull request merged**